jobs/JRRFS_UNGRIB

```diff
@@ -27,26 +27,28 @@ export GROUP_TOTAL_NUM=${GROUP_TOTAL_NUM:-1}
 export pid=${pid:-$$}
 if [[ -z "${ENS_INDEX}" ]]; then
   export RUN="rrfs"
+  export SUBDIR=""
 else
   export RUN="ensrrfs"
+  export SUBDIR="/m${ENS_INDEX}"
 fi
```

Comment:
We need to rethink how to make RUN and SUBDIR. Should add WGF as part of name, like

RUN="rrfs"${WGF}

Reply:
Also, should use WGF to decide how to set up SURDIR?

Reply:
do we need SUBDIR?

Reply:
Yes, we do need it to have different directories for the members.